await.js

```diff
@@ -21,11 +21,23 @@ const readFile = (options, filename) =>
 
 async function main() {
   const index = path.join(process.argv[2], 'index.txt');
-  const data = await readFile({encoding: 'utf8'}, index);
+  let data;
+  try {
+    data = await readFile({encoding: 'utf8'}, index);
+  } catch (err) {
+    process.stderr.write(String(err) + '\n');
+    process.exit(1);
+  }
```

Comment:
If you omit the `try/catch` won't this result in an uncaught exception that crashes the program with a non-zero exit code?